doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
   selecting text on the page, or \c{false} otherwise.
 */
 
+/*!
+  \qmlproperty bool WebView::downloadsEnabled
+  \brief Whether downloads are enabled within the webview
+
+  If downloads are enabled, long-press context menus will include options to
+  save the link or image to the device, when appropriate.
+*/
+
+/*!
+  \qmlproperty bool WebView::canShowSelectionMarkers
+  \brief Whether text selection markers will be shown
+
+  If selection markers are allowed to be shown, draggable text selection
+  markers will be shown when the user selects text from a document
+  shown in the webview.
+*/
+
+/*!
+  \qmlproperty bool WebView::textSelectionActive
+  \readonly
+  \brief Whether the user is currently selecting text
+
+  While text selection is active, the text selection controller will
+  be receiving touch input events (e.g. to allow the selection markers
+  to be dragged), rather than the webview itself (thus preventing
+  accidental panning etc).
+*/
+
+/*!
+  \qmlproperty Item WebView::textSelectionController
+  \brief The item which handles text selection gestures
+
+  If the client application doesn't explicitly provide a text selection
+  controller, a default TextSelectionController from Sailfish.WebView.Controls
+  will be constructed and used when required.
+
+  Any text selection controller provided by the client must conform
+  to the required API.  See the documentation for\c{Sailfish.WebView.Controls}
+  for more information.
+*/
+
+/*!
+  \qmlmethod WebView::clearSelection()
+  \brief Clears any text currently selected by the user.
+*/
+
+/*!
+  \qmlsignal WebView::linkClicked(string url)
+  \brief This signal is emitted when the user selects a link on the page.
+
+  When emitted \a url will be set to href selected.
+*/
+
+// ------------------------------------------------------ from RawWebView.qml:
+
+/*!
+  \qmlproperty qreal WebView::virtualKeyboardMargin
+  \brief The margin taken by the keyboard at the bottom of the page
+
+  When the keybaord is opened it may overlap web content displayed in a
+  \l{WebView}. Setting this value allows the web content to be arranged to
+  compensate for the overlapped area.
+
+  When using a \l{WebView} this property will be set automatically when the
+  keyboard is opened or closed.
```

Comment:
Maybe putting even more emphasis on the fact that developer usually do not need to change the default behaviour.

Reply:
done